### PR TITLE
Change the name of virtual assistant

### PR DESCRIPTION
**Pull request by Tejaso7**
hey,There I have successfully Added the functionality of Change name of VA please check this

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -57,7 +58,7 @@ def respond(voice_data):
         webbrowser.get().open(url)
         alina_speak(f"Here is what I found for {search}")
     if "what is your name" in voice_data:
-        alina_speak("My name is Tejax.")
+        alina_speak("My name is alina.")
```
Comment (spignelon) [inline main.py]: You've created a new variable assistant_name and added the feature to change it using the voice, but you forgot to replace the name from the string and using the assistant_name variable here instead.

Comment (spignelon) [reply main.py]: commit this small change and I'll merge the Pull Request right away.
Nice work

**Comment by spignelon** [comment]
Also consider removing redundant commits such as: https://github.com/spignelon/virtual-assistant/pull/26/commits/b46746c92eb416b93e1a0941c20dd2ec5266cdd9 and https://github.com/spignelon/virtual-assistant/pull/26/commits/1eab59616f7ecfc4f317de3a066f0d1393ae1395 which doesn't add, contribute or change anything

**Comment by Tejaso7** [comment]
check now

**Comment by Tejaso7** [comment]
Check out ✅ if any changes... Mention me back

**Comment by spignelon** [comment]
> Check out ✅ if any changes... Mention me back

Nothing is really changed, the previous issue and review still applies https://github.com/spignelon/virtual-assistant/pull/26#discussion_r1369866965 and https://github.com/spignelon/virtual-assistant/pull/26#issuecomment-1776849054

**Pull request closed without merging**